packages/terra-framework-docs/src/terra-dev-site/test/menu/menu/CustomIconsMenu.test.jsx

```diff
@@ -35,40 +41,40 @@ class CustomIconsMenu extends React.Component {
   render() {
     return (
       <div>
-        <Menu
-          isOpen={this.state.open}
-          targetRef={this.getButtonNode}
-          onRequestClose={this.handleRequestClose}
-        >
-          <Menu.Item
-            text="Error Icon"
-            key="Error"
-            icon={<IconError />}
-          />
-          <Menu.Item
-            text="Alert Icon"
-            key="Alert"
-            icon={<IconAlert />}
-          />
-          <Menu.Item
-            text="Warning Icon"
-            key="Warning"
-            icon={<IconWarning />}
-          />
-          <Menu.Item
-            text="Help Icon"
-            key="Help"
-            icon={<IconHelp />}
-          />
-          <Menu.Item
-            text="Required Icon"
-            key="Required"
-            icon={<IconRequired />}
-          />
-        </Menu>
-        <button type="button" id="custom-icons-button" onClick={this.handleButtonClick} ref={this.setButtonNode}>
-          Custom Icons Menu
-        </button>
+        <div className={cx('menu-wrapper')} ref={this.setButtonNode}>
```

Comment:
```suggestion
        <div className={cx('menu-wrapper')} ref={this.setButtonNode}>
```
same here as well !! I believe it should be on button

Reply:
Updated button ref in https://github.com/cerner/terra-framework/pull/1626/commits/0e944cfad9d2cb5c81725ab373c191e60248a11f